src/HttpHandler/HttpHandlerFactory.php

```diff
@@ -30,7 +33,17 @@ class HttpHandlerFactory
      */
     public static function build(ClientInterface $client = null)
     {
-        $client = $client ?: new Client();
+        if (is_null($client)) {
+            $stack = null;
+            if (class_exists(BodySummarizer::class)) {
+                // double the # of characters before truncation by default
+                $bodySummarizer = new BodySummarizer(240);
```

Comment:
Should this be made configurable somehow? Is there already a way to configure things in this package?

Reply:
It's possible for the end user to pass in custom guzzle clients like this:
```php
$http = new GuzzleHttp\Client($someCustomConfig);
$assetClient = new AssetServiceGapicClient([
    'credentialsConfig' => [
        'authHttpHandler' => $http, // for calls made to the token endpoint
    ],
    'rest' => [
        'httpHandler' => $http,     // for calls made to the API using REST
    ]
]);
```

However, this change is to modify the default value, so we're just hardcoding it.